<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cleaning Schedule Generator</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select starting week</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Generate schedule</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Schedule"/>
    <addaction name="actionOpen_Schedule"/>
    <addaction name="actionSave_as_pdf"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Cleaning_schedule_generator"/>
   </widget>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>Schedule</string>
    </property>
    <widget class="QMenu" name="menuRoom">
     <property name="title">
      <string>Rooms</string>
     </property>
     <addaction name="actionAdd_Room"/>
     <addaction name="actionRemove_Room"/>
    </widget>
    <addaction name="actionGenerate"/>
    <addaction name="separator"/>
    <addaction name="menuRoom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSchedule"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionAbout_Cleaning_schedule_generator">
   <property name="text">
    <string>About CSG</string>
   </property>
   <property name="toolTip">
    <string>About Cleaning Schedule Generator</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionRemove_Room">
   <property name="text">
    <string>Remove Room</string>
   </property>
  </action>
  <action name="actionAdd_Room">
   <property name="text">
    <string>Add Room</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSave_as_pdf">
   <property name="text">
    <string>Save as .pdf</string>
   </property>
  </action>
  <action name="actionNew_Schedule">
   <property name="text">
    <string>New Schedule</string>
   </property>
  </action>
  <action name="actionOpen_Schedule">
   <property name="text">
    <string>Open Schedule</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
